src/main/java/com/cloudbees/jenkins/support/util/Helper.java

```diff
@@ -0,0 +1,27 @@
+package com.cloudbees.jenkins.support.util;
+
+import jenkins.model.Jenkins;
+import javax.annotation.Nonnull;
+
+/**
+ * Simple helper so we don't have to check {@code Jenkins.getInstance() != null} everywhere.
+ *
+ * TODO: replace with Jenkins.getActiveInstance() when on core {@literal >=} 1.609
```

Comment:
What? Already on 2.46 in pom.xml: https://github.com/jenkinsci/support-core-plugin/blob/70684fc409f53a10db0c0d31c58fc3dbdf82d65f/pom.xml#L72